task7/js/script.js

```diff
@@ -0,0 +1,560 @@
+class AdModel{
+
+    constructor (adList){
+        if(adList.length != 0){
+            this._adList = adList.slice();
+        }else{
+            this._adList = [];
```

Comment:
You can use empty array as a default parameter of constructor 
```
constructor (adList = [])
```
Then you won't need to check length of `adList`